test/junit/scala/math/DoubleTest.scala

```diff
@@ -0,0 +1,16 @@
+package scala.math
+
+import org.junit.Assert._
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+@RunWith(classOf[JUnit4])
+class DoubleTest {
+
+  /* Test for scala/bug#11386 */
+  @Test
```

Comment:
May I suggest testing also plus/minus infinities and zeroes?

Reply:
Done in 430c7acb33d41ea95360f613fe9794387fd488b8